runtimes/core/src/lib.rs

```diff
@@ -52,8 +53,61 @@ pub mod encore {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct ProcessConfig {
```

Comment:
Maybe move this out of `lib.rs` into a separate file. Can use `pub use` to re-export from here